sdl_android/src/main/java/com/smartdevicelink/proxy/RPCRequestFactory.java

```diff
@@ -841,6 +841,7 @@ public static SubscribeVehicleData BuildSubscribeVehicleData(boolean gps, boolea
 		msg.setExternalTemperature(externalTemperature);
 		msg.setPrndl(prndl);
 		msg.setTirePressure(tirePressure);
+		msg.setEngineOilLife(engineOilLife);

```

Comment:
Please remove all changes to this file. We are no longer supporting the class. Also changing method signatures is a breaking change